src/ConnectedMode/Resources.resx

```diff
@@ -242,4 +242,25 @@
   <data name="Suppression_IssueChangedEvent" xml:space="preserve">
     <value>[Suppressions] Handling IssueChangedEvent, isResolved: {0}, issue keys for current branch: {1}</value>
   </data>
+  <data name="ImportBeforeFileGenerator_CheckingIfFileExists" xml:space="preserve">
+    <value>[ConnectedMode/ImportsBeforeFileGenerator] Checking if file SonarLint before imports targets file exists</value>
```

Comment:
I'd probably include the full path to the file being checked in this message to help us with debugging/support e.g.

`[ConnectedMode/ImportsBeforeFileGenerator] Checking if file SonarLint before imports targets file exists. File: {foo}`